packages/middleware-sdk-s3/src/throw-200-exceptions.spec.ts

```diff
@@ -15,58 +16,81 @@ describe("throw200ExceptionsMiddlewareOptions", () => {
     jest.clearAllMocks();
   });
 
-  it("should throw if response body is empty", async () => {
-    expect.assertions(3);
+  describe("tests for statusCode < 200 and >= 300", () => {
     mockStreamCollector.mockResolvedValue(Buffer.from(""));
     mockUtf8Encoder.mockReturnValue("");
-    mockNextHandler.mockReturnValue({
-      response: new HttpResponse({
-        statusCode: 200,
-        headers: {},
-        body: "",
-      }),
-    });
-    const handler = throw200ExceptionsMiddleware(mockConfig)(mockNextHandler, {} as any);
-    try {
-      await handler({
+
+    it.each([199, 300])("results for statusCode %i", async (statusCode) => {
+      mockNextHandler.mockReturnValue({
+        response: mockResponse,
+        statusCode,
+        // headers: {},
+        // body: "",
```

Comment:
Uncomment, and pass body with some content?
```suggestion
        headers: {},
        body: "body",
```